ArcGISRuntimeSDKQt_CppSamples/Scenes/DisplayScenesInTabletopAR/README.md

```diff
@@ -0,0 +1,48 @@
+# Display scenes in tabletop AR
+
+Use augmented reality (AR) to pin a scene to a table or desk for easy exploration.
+
+![](screenshot.png)
```

Comment:
I took a look at the screenshot. Can you try and get one zoomed out a little more that makes it clear what is going on? The current one looks like any other scene layer and is hard to tell it is in AR